src/components/v2/shared/FundingCycleConfigurationDrawers/NftDrawer/index.tsx

```diff
@@ -211,6 +224,21 @@ export default function NftDrawer({
           >
             <NftMarketplaceCustomizationForm form={marketplaceForm} />
           </MinimalCollapse>
+          <MinimalCollapse
+            header={
+              <>
+                <Trans>Purchase confirmation popup</Trans>
+                <TooltipIcon
+                  tip={t`Show your contributors a message after they buy an NFT.`}
```

Comment:
**question** Are we referring to nft rewards as "buying NFT"? I don't think we should probably.

`After they receive their NFT reward` might be better/safer.

Reply:
Yep good call.